feat(content): Culture Conversation for the 300th Anniversary of the Republic (#10640)
endless-sky/endless-sky: data/human/culture conversations.txt

			`Watching the microgravity section of the spaceport from one of the bars in Mordente Cylinder, you observe a Bastion in pirate markings coming nearly too fast for safe approach to the station, its hull covered in gashes and burns from what looks to be Plasma Cannon shots.`
			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs. Meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`
			`	Within the hour the work shuttles scatter from the ship as they finish patching up its battered hull, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`

> mission "Republic 300 Years Anniversary"

Reviewer: ```suggestion
mission "Republic 300 Year Anniversary"
```

> mission "Republic 300 Years Anniversary"

Reviewer: ```suggestion
mission "Republic 300 Year Anniversary"
```

	landing
	source
		government "Republic"

> 		attributes "spaceport"

Reviewer: ```suggestion
		not attributes "uninhabited"
```
Some PRs might add new stations that refuel you but are still uninhabited.

	to offer
		year == 3030
	on offer
		conversation

> 			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport, in something of a celebration as they tune in to the main news stations, websites and blogs throughout human space that are commenting on this momentous day. Just as well, however, you notice many passersby moving about with an air of disdain for the activities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`

Reviewer: ```suggestion
			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport in something of a celebration as they tune in to the main news stations, websites and blogs throughout human space that are commenting on this momentous day. Just as well, however, you notice many passersby moving about with an air of disdain for the activities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`
```
Also, why "just as well"? Does the player dislike the Republic?

> 			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport in something of a celebration as they tune in to the main news stations, websites and blogs throughout human space that are commenting on this momentous day. Just as well, however, you notice many passersby moving about with an air of disdain for the activities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`

Reviewer: Alright, so then:
I think using just "However" works here, because it does make contrast, while not giving negative implications.
```suggestion
			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport in something of a celebration as they tune in to the main news stations, websites and blogs throughout human space that are commenting on this momentous day. However, you notice many passersby moving about with an air of disdain for the festivities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`
```

> 			`	Originally formed by Earth and planets close to it, the Rim, and a few Dirt Belt worlds, the Republic went on to gradually expand, incorporating the Syndicate, Paradise Worlds, and the Deep. For three hundred years now, most of humanity has existed under its flag, and through protests and the space renaissance, waves of colonization, expansion, and battles against remnants of the Alphas, that flag remains flying over most human worlds today.`

Reviewer: Last sentence feels somewhat run-on, even though it's technically correct.

Author: Idk I think it works

Reviewer: Alright. It still can not make sense to me if I think about it the wrong way, but I don't really know how to split it up.

			branch "war over"
				has "main plot completed"

> 			`	Not all is well, of course, as the bombing of Geminus and Martini appears to have set the Republic on the precipice of war with the Free Worlds, on the southern portion of human space, and you can't help but feel that the cheerfulness in the spaceport celebration has more to do with the national holiday, than with a deep-rooted appreciation for the Republic's history, and what it stands for.`

Reviewer: One sentence?

Author: Split it at the middle.

Reviewer: Looks good now.

				goto end
			label "war over"

> 			`	The recent Free Worlds war seemed to threaten this long history of stability, as battles raged over the skies of dozens of planets, and the threat of an open nuclear conflict loomed closer as the conflict escalated. Alas, the indiscriminate disappearance of many systems close to Earth, followed by the unified human front against the Pug invaders, seems to have knit together the sentiments of many throughout the Republic - for the time being, at least.`

Reviewer: ```suggestion
			`	The recent Free Worlds war seemed to threaten this long history of stability, as battles raged over the skies of dozens of planets, and the threat of an open nuclear conflict loomed closer as the conflict escalated. The indiscriminate disappearance of many systems close to Earth, followed by the unified human front against the Pug invaders, seems to have knit together the sentiments of many throughout the Republic - for the time being, at least.`
```
Is it sad that humans united?

			branch "zenith bombed"
				has "Wanderers: Sestor: Return to Wanderers: done"
			branch "sestor beaten at farpoint"
				has "Wanderers: Sestor Alt: Return to Wanderers: done"
			label end

> 			`	You decide to take an hour or two to join in on the festivities, it's not everyone who can claim to have witnessed a centenary anniversary of a nation, after all. Stepping off the <ship>, you head to the spaceport, having a couple of drinks, speaking to some of the locals, and partying with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`

Reviewer: I think this sounds better.
```suggestion
			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, having a couple of drinks, speaking to some of the locals, and partying with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`
```

				decline
			label "zenith bombed"

> 			`	Dimming the mood of the otherwise mostly positive news titles, are the comments about a mysterious armada of robotic warships invading the Far North. Having crushed the Navy forces over Farpoint, the fleet was only partially beaten after the detonation of a terrible bomb on Zenith, which wrecked much of the planet. Indeed, it was weeks still before the scattered ships were fully destroyed, and confidence in the Navy's - and by extension the Republic's - ability of keeping humanity safe seems to have been affected significantly...`

Reviewer: ```suggestion
			`	Dimming the mood of the otherwise mostly positive news titles are the comments about a mysterious armada of robotic warships invading the Far North. Having crushed the Navy forces over Farpoint, the fleet was only partially beaten after the detonation of a terrible bomb on Zenith, which wrecked much of the planet. Indeed, it was weeks still before the scattered ships were fully destroyed, and confidence in the Navy's - and by extension the Republic's - ability of keeping humanity safe seems to have been affected significantly...`
```

				goto end
			label "sestor beaten at farpoint"
			`	Web comments and news anchors seem to be particularly excited to comment on the recent victory of the Navy over the mysterious armada of robotic warships that invaded the Far North. Though at a great cost to the northern fleet garrisons, the invading fleets were successfully stopped at the Farpoint base, preventing the ships from causing any further harm. Veterans of the Farpoint battle seem to be hoarding the spotlight, giving emotional reports of their memory of the fight, and of the sacrifices they went through to achieve victory.`

> 				goto end

Reviewer: ```suggestion
			label end
```
👀

Author: This is a goto so that it returns to the `end` label and declines properly, as the `end` label is in a line before this one.

			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs. Meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`
			`	Within the hour the work shuttles scatter from the ship as they finish patching up its battered hull, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`

> 

Reviewer: ```suggestion
```

	landing
	source
		government "Republic"
		not attributes "uninhabited"
	to offer
		year == 3030
	on offer
		conversation
			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport in something of a celebration as they tune in to the main news stations, websites, and blogs throughout human space that are commenting on this momentous day. However, you notice many passersby moving about with an air of disdain for the festivities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`
			`	Originally formed by Earth and planets close to it, the Rim, and a few Dirt Belt worlds, the Republic went on to gradually expand and incorporate the Syndicate, Paradise Worlds, and the Deep. For three hundred years now, most of humanity has existed under its flag, and through protests and the space renaissance, waves of colonization, expansion, and battles against remnants of the Alphas, that flag remains flying over most human worlds today.`
			branch "war over"
				has "main plot completed"

> 			`	Not all is well, of course, as the bombing of Geminus and Martini appears to have set the Republic on the precipice of war with the Free Worlds, in the southern portion of human space. You can't help but feel that the cheerfulness of the spaceport celebration has more to do with the national holiday than with a deep-rooted appreciation for the Republic's history and what it stands for.`

Reviewer 2: What if I get this in the middle of the FW war? In that case the Republic isn't on the precipice of war, it's knee-deep in it.

Author: Mmm good point. Would it be preferrable to add another branch, or prevent this from offering during the war?

Reviewer 2: I'd prefer adding another branch if you can do that.

				goto end
			label "war over"

> 			`	The recent Free Worlds war seemed to threaten this long history of stability, as battles raged over the skies of dozens of planets, and the threat of an open nuclear conflict loomed closer as the conflict escalated. The indiscriminate disappearance of many systems close to Earth, followed by the unified human front against the Pug invaders, seems to have knit together the sentiments of many throughout the Republic - for the time being, at least.`

Reviewer 2: ```suggestion
			`	The recent Free Worlds war seemed to threaten this long history of stability, as battles raged over the skies of dozens of planets, and the threat of an open nuclear war loomed closer as the conflict escalated. The indiscriminate disappearance of many systems close to Earth, followed by the unified human front against the Pug invaders, seems to have knit together the sentiments of many throughout the Republic - for the time being, at least.`
```

			branch "zenith bombed"
				has "Wanderers: Sestor: Return to Wanderers: done"
			branch "sestor beaten at farpoint"
				has "Wanderers: Sestor Alt: Return to Wanderers: done"
			label end

> 			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, having a couple of drinks, speaking to some of the locals, and partying with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`

Reviewer 2: ```suggestion
			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, have a couple of drinks, speak to some of the locals, and party with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`
```

> 			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, having a couple of drinks, speaking to some of the locals, and partying with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`

Reviewer 2: Perhaps add a choice here.

Author: What would the choice entail?

Reviewer 2: Whether or not to join the celebrations.

Author: Tried my hand at the branch and choice, lmk if anything needs updating still.

	to offer
		year == 3030
	on offer
		conversation
			`News headlines bombard your comms as you land on <origin>: today is the three hundredth anniversary of the Republic. A crowd has gathered in the local spaceport in something of a celebration as they tune in to the main news stations, websites, and blogs throughout human space that are commenting on this momentous day. However, you notice many passersby moving about with an air of disdain for the festivities, simply trying to get their business here over quickly, and being frustrated at the inevitable slowness of the local facilities today.`
			`	Originally formed by Earth and planets close to it, the Rim, and a few Dirt Belt worlds, the Republic went on to gradually expand and incorporate the Syndicate, Paradise Worlds, and the Deep. For three hundred years now, most of humanity has existed under its flag, and through protests and the space renaissance, waves of colonization, expansion, and battles against remnants of the Alphas, that flag remains flying over most human worlds today.`
			branch "war over"
				has "main plot completed"
			branch "war ongoing"
				has "chosen sides"
				not "main plot completed"

> 			`	Not all is well, of course, as the bombing of Geminus and Martini appears to have set the Republic on the precipice of war with the Free Worlds, in the southern portion of human space. You can't help but feel that the cheerfulness of the spaceport celebration has more to do with the national holiday than with a deep-rooted appreciation for the Republic's history and what it stands for.`

Reviewer 3: The last part feels like it'd very location dependent. Maybe in the dirt belt worlds, particularly the ones that join the FW early in the war, it makes sense, but I imagine most people in the Deep or Paradise Worlds are pretty happy with the Republic.

Reviewer 3: @Arachi-Lover ?

Author: Was changed to `	Not all is well, of course, as the bombing of Geminus and Martini appears to have set the Republic on the precipice of war with the Free Worlds, in the southern portion of human space. You can't help but feel that much of the cheerfulness about this date has more to do with the national holiday than with a deep-rooted appreciation for the Republic's history and what it stands for.` , the intent being to keep the overall message but remove the specific mention of the local celebration, so it would be referring to a more general feeling/reception of the date from media and online means.
Is it still off?

				goto end
			label "war over"
			`	The recent Free Worlds war seemed to threaten this long history of stability, as battles raged over the skies of dozens of planets, and the threat of an open nuclear war loomed closer as the conflict escalated. The indiscriminate disappearance of many systems close to Earth, followed by the unified human front against the Pug invaders, seems to have knit together the sentiments of many throughout the Republic - for the time being, at least.`
			branch "zenith bombed"
				has "Wanderers: Sestor: Return to Wanderers: done"
			branch "sestor beaten at farpoint"
				has "Wanderers: Sestor Alt: Return to Wanderers: done"
			label end
			choice
				`	(Celebrate the Republic's 300th anniversary.)`
					goto celebrate
				`	(Go on about your day as usual.)`

> 			`	You switch your media channels off from the celebrations, and get to your ship's system checks. Once you're done, you spend the rest of the day in the cockpit, thinking up the next course for you to chart, and head to your bunk at night, for the end of just another day.`

Reviewer 2: ```suggestion
			`	You switch your media channels off from the celebrations, and get to your ship's system checks. Once you're done, you spend the rest of the day in the cockpit, thinking up the next course for you to chart. You head to your bunk once night falls, the end of just another day.`
```

Reviewer 3: The thing about staying in the cockpit all day and then going to bed doesn't really work if, well, I do anything else, or if anything happens to me.

				decline
			label celebrate

> 			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, have a couple of drinks, speak to some of the locals, and party with a few revelers. Once it's getting late, you head back to your ship, and reflect on the meaning of this day throughout human space.`

Reviewer 2: ```suggestion
			`	You decide to take an hour or two to join in on the festivities, because after all, it's not everyone who can claim to have witnessed a centenary anniversary of a nation. Stepping off the <ship>, you head to the spaceport, have a couple of drinks, speak to some of the locals, and party with a few revelers. Once it's getting late, you head back to your ship and reflect on the meaning of this day throughout human space.`
```

Reviewer 3: I think "You decide to take [...]" ought to be changed to "You take [...]". I did indeed decide, but I don't need to be told that I decided.

				decline
			label "zenith bombed"

> 			`	Dimming the mood of the otherwise mostly positive news titles are the comments about a mysterious armada of robotic warships invading the Far North. Having crushed the Navy forces over Farpoint, the fleet was only partially beaten after the detonation of a terrible bomb on Zenith, which wrecked much of the planet. Indeed, it was weeks still before the scattered ships were fully destroyed, and confidence in the Navy's - and by extension the Republic's - ability of keeping humanity safe seems to have been affected significantly...`

Reviewer 3: ```suggestion
			`	Dimming the mood of the otherwise mostly positive news titles are the comments about a mysterious armada of robotic warships invading the Far North. Having crushed the Navy forces over Farpoint, the fleet was only partially beaten after the detonation of a terrible bomb on Zenith, which wrecked much of the planet. Indeed, it was weeks still before the scattered ships were fully destroyed, and confidence in the Navy's - and, by extension, the Republic's - ability to keep humanity safe seems to have been affected significantly...`
```
